app/scripts/metamask-controller.js

```diff
@@ -2546,6 +2547,23 @@ export default class MetamaskController extends EventEmitter {
     }
   }
 
+  /**
+   * Sets the Ledger Live preference to use for hardware wallet support
+   * @param {bool} val - the value representing if they want Ledger Live support
+   * @param {Function} cb - A callback function called when complete.
+   */
+  setLedgerLivePreference(val, cb) {
+    try {
+      this.preferencesController.setLedgerLivePreference(val)
+      cb(null)
+      return
+    } catch (err) {
+      cb(err)
+      // eslint-disable-next-line no-useless-return
```

Comment:
Nit: We should follow this rule, rather than disable it

Reply:
Not sure why this was marked as resolved :thinking:

Reply:
Sorry, I likely marked it as resolved because we're using this pattern throughout the controller.  I can remove it but it seems like we need a larger effort to remove it completely.

Reply:
Oh I see - I hadn't realized. I mention because I've seen this used before because of an apparent conflict between this rule and the consistent return rule. But it has an easier workaround in cases like this where you want to return nothing: `return undefined`